hub/core/storage/lru_cache.py

```diff
@@ -107,6 +107,17 @@ def __delitem__(self, path: str):
             if not deleted_from_cache:
                 raise
 
+    def clear_cache(self):
+        """Delete the contents of all layers of the cache but not from the final storage."""
+        self.cache_storage.clear()
+        if self.next_storage.hasattr("clear_cache"):
+            self.next_storage.clear_cache()
+
+    def clear(self):
+        """Deletes all the data from all the layers of the cache and the final storage."""
```

Comment:
The difference between these two is pretty drastic, I think we should reword this docstring just so they don't look so similar. Or at least a note that this is irreversible or something.